values.schema.json

```diff
@@ -1163,11 +1163,16 @@
                     "default": null
                 },
                 "MaxNoOfConcurrentOperations": {
-                    "type": [
-                        "integer",
-                        "null"
-                    ],
-                    "default": null
+                    "type": "integer",
+                    "default": 32768,
```

Comment:
you need to change the default as well here since that will override the values when publishing this